crates/swift-bridge-ir/src/codegen/codegen_tests/option_codegen_tests.rs

```diff
@@ -846,7 +846,10 @@ mod shared_struct_with_option_field_ffi_repr {
         ExpectedSwiftCode::ContainsAfterTrim(
             r#"
 public struct SomeStruct {
-    var field: Optional<UInt8>
+    public init(field: Optional<UInt8>) {
```

Comment:
Super minor.. Can we put the initializer after the fields and newline in between them?

My thinking here is that it's nice for anyone that's looking at a struct definition to be able to see the most important thing (what fields are available) first.